### PR TITLE
TELCODOCS-960 - PTP config `phc2sysOpts` clarify use of the -m switch

### PR DESCRIPTION
**Pull request by aireilly**
https://issues.redhat.com/browse/TELCODOCS-960

In the section 'Intel Columbiaville E800 series NIC as PTP ordinary clock reference' the `phc2sysOpts` recommended command line `-a -r -m -n 24 -N 8 -R 16` contains the `-m` option which means all output will go to stdout (console). This is confusing to users. 

Added a note that explains this `phc2sysOpts` value. `-m` prints messages to `stdout` so that logs can be parsed to generate meaningful Prometheus metrics.

Preview:
* https://50901--docspreview.netlify.app/openshift-enterprise/latest/networking/using-ptp.html#cnf-configuring-the-ptp-fast-event-publisher_using-ptp
* https://50901--docspreview.netlify.app/openshift-enterprise/latest/networking/using-ptp.html#configuring-linuxptp-services-as-ordinary-clock_using-ptp
* https://50901--docspreview.netlify.app/openshift-enterprise/latest/networking/using-ptp.html#nw-columbiaville-ptp-config-refererence_using-ptp
* https://50901--docspreview.netlify.app/openshift-enterprise/latest/networking/using-ptp.html#cnf-configuring-the-ptp-fast-event-publisher_using-ptp

Merge to main, CP to enterprise-4.9+

### CODE REVIEW

**Comment by aireilly** [comment]
Closing. This bug was opened in error.

**Comment by aireilly** [comment]
Reopening to add docs clarification

**Review by aneeshkp — APPROVED** [review]

File: modules/cnf-configuring-cvl-nic-as-oc.adoc [hunk 1]
```diff
@@ -37,9 +37,9 @@ spec:
   - name: "slave"
     interface: <interface_name> <1>
     ptp4lOpts: "-2 --summary_interval -4" <2>
-    phc2sysOpts: "-a -r -m -n 24 -N 8 -R 16"
-    ptpSchedulingPolicy: "SCHED_FIFO" <3>
-    ptpSchedulingPriority: 65 <4>
+    phc2sysOpts: "-a -r -m -n 24 -N 8 -R 16" <3>
```
Comment (kelbrown20) [inline modules/cnf-configuring-cvl-nic-as-oc.adoc:40]: The changes in this module were not showing up for me in the preview. Then I noticed that this module is not in the assembly that is listed in line 3
https://github.com/openshift/openshift-docs/blob/main/networking/using-ptp.adoc
This shows why the preview isn't rendering.

Is that right @aireilly or is there something I'm missing here? Or maybe the module accidentally get removed at some point

**Review by kelbrown20 — COMMENTED** [review]
Great work! Other than that one thing, This LGTM!

**Comment by obochan-rh** [comment]
Perfect /LGTM

**Pull request merged**